### ISSUE TITLE
doubts about text-guided encoding

### ISSUE THREAD

**Issue by fengxinmin** [effl-lab/TACO#2]
Thank you for opening the source code of this work. Basically, I found that the released models do indeed achieve the performance claimed in the paper. However, I have serious doubts about whether the text truly helps with compression, for two main reasons:

- During training, random cropping can lead to discrepancies between the text and the image. For example, for an image with the caption "a boat in the water next to a pier," random cropping during training may result in the region of the boat not being fed into the neural network, which is likely quite common. 
- I used your released weights and fixed all input text to "NA", and the results did not show any obvious differences. This suggests that the text does not aid in encoding; rather, it is the loss functions and the additional model complexity introduced by the text adapter that are truly beneficial for encoding.

Based on the above reasons, I think that this paper does not fulfill the claimed text-guided encoding, which may mislead other readers.

**Comment by fengxinmin** [comment]
![performance](https://github.com/user-attachments/assets/f4df2686-8791-427d-b4e4-8c2ec5392c4c)

**Comment by minguinho26** [comment]
Hi, thank you for taking the time to read our paper and asking questions. 
The responses to your questions are like below.

Q. Random cropping can lead to discrepancies between the text and the image
A. As you noted, random cropping can reduce the correlation with the caption. However, cropped images still retain some of the text's semantic information. Like you, we identified the limitations of random cropping and considered resizing as an alternative to preserve image-caption correspondence. However, resizing distorts pixel distribution, which could negatively affect the training of image compression. Therefore, we opted for random cropping to maintain batch training while minimizing corruption with respect to pixel reconstruction in the training process.

Q. The text does not aid in encoding.
A. As you pointed out, the choice of text during inference shows slight performance differences, which may lead to doubts about text-guided encoding. However, experimental results clearly show that using text guidance during training improves performance, and we confirmed that using text in encoding only (i.e. text-guided encoding) achieves the best results. Thus, we said the contribution of text-guided encoding in our paper.

I hope my answer is helpful.

Thank you,
Minkyu Kim.
